lib/components/ConflictPicker.vue

```diff
@@ -49,17 +50,35 @@
 
 		<!-- Controls -->
 		<template #actions>
-			<NcButton data-cy-conflict-picker-skip @click="onSkip">
+			<!-- Cancel the entire operation -->
+			<NcButton :aria-label="t('Cancel')"
```

Comment:
aria label does not need to be set when a text is set.

Reply:
According to Grigorii's compoent (which I cannot use right now as it's requiring a callback prop), yes, we set both 
https://github.com/nextcloud-libraries/nextcloud-vue/blob/9c44ebc68bdecbe1d653c44745c23e139f80c694/src/components/NcDialogButton/NcDialogButton.vue#L28-L29 :shrug:

Reply:
As long as the text does not differ it does not matter, so ok :)